.gitignore

```diff
@@ -8,4 +8,5 @@ build/
 content/
 i18n/
 !i18n/en/
-docs/v**
\ No newline at end of file
+docs/v**
+erick
```

Comment:
Nit, but do you want to include `erick` in the ignore? Wasn't sure if that was just for testing

Reply:
lol